test/test_mixed_UKF_KF/main.cpp

```diff
@@ -55,11 +57,15 @@ class MixedUKFKFSimulation : public GaussianFilter
 };
 
 
-int main()
+int main(int argc, char* argv[])
 {
     std::cout << "Running a mixed UKF/KF filter on a simulated target." << std::endl;
-    std::cout << "Data is logged in the test folder with prefix testUKF_KF." << std::endl;
 
+    const bool write_to_file = (argc > 1 ? std::string(argv[1]) == "ON" : false);
+    if (write_to_file)
+        std::cout << "Data is logged in the test folder with prefix testSIS." << std::endl;
```

Comment:
I think it should be `... with prefix testUKF_KF`.

Reply:
Done!